national-medical-journal-of-india.csl

```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" page-range-format="chicago" demote-non-dropping-particle="sort-only" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>The National Medical Journal of India</title>
+    <id>http://www.zotero.org/styles/national-medical-journal-of-india</id>
```

Comment:
@damnation333, could you add "the-" to the file name, and style ID and "self" link.

(when in doubt whether "The" is part of a journal's title, I usually check the cover image of the journal, e.g. at http://www.nmji.in/)